styles.css

```diff
@@ -0,0 +1,11 @@
+* {
+  box-sizing: border-box;
```

Comment:
Maybe you want to add *::before and *::after here?

Reply:
Done, thanks for your feedback 👍